calibration/NUTNRM/CGINS-NUTNRM-00711__20250312.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+711,CC_cal_temp,20.00,source_file 3305-00327-00257-A_SN_0711_Pre-Deployment_NUTNR-M.zip > SNA0711D.CAL; reference update conducted in Clark South Cold Room at 12C
```

Comment:
the pre-deploys above have the prefix NTR- before SN in 1st column, e.g., NTR-1103

Reply:
the glider/auv ones do not have NTR as part of SN